### ISSUE TITLE
Consider renaming `VRDisplay#getEyeParameters(VREye)` to `VRDisplay#eyes` array

### ISSUE THREAD

**Issue by cvan** [w3c/webvr#183]
See [OSVR's `hmd.eyes` array](https://github.com/OSVR/OSVR-JSON-Schemas/blob/545fc02/display_descriptor_schema_v1.json#L179). Proposed sample usage:

```js
vrDisplay.eyes;  // [<VR Eye offset="0.5" renderWidth="…" renderHeight="…">, <VR Eye offset="0.5" renderWidth="…" renderHeight="…">]
```

**Comment by toji** [comment]
It's been my intention to get rid of `VREyeParameters` entirely, since it's mostly filled with values that could either be misused or should be delivered another way.

* `offset` is baked into the view matrices
* `fieldOfView` is baked into the projection matrices
* `canvasWidth/Height` should be more flexible, allowing devs to choose quality over speed.

**Comment by toji** [comment]
This is effectively resolved in the [2.0 proposal](https://github.com/w3c/webvr/blob/master/explainer.md#appendix-b-proposed-idl) so I'm closing this issue.
